.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,29 @@
+name: Feature request
+description: Request a new feature to be added to OpenRCT2.
+labels:
+- feature
+
+body:
+- type: markdown
+  attributes:
+    value: Before filing this request, [please check if it hasn't already been suggested before](https://github.com/OpenRCT2/OpenRCT2/issues?q=is%3Aissue+is%3Aopen+label%3Afeature). If it has, please add to the existing issue instead.
```

Comment:
Can we change the filter to include both open + closed issues? Because the suggestion might have been something that we already discarded.